src/main/java/net/datafaker/service/FakerIDN.java

```diff
@@ -17,16 +17,16 @@ public static String toASCII(String in) {
         } catch (Exception e) {
             // let's continue with the character by character encoding hack.
         }
-        final StringBuilder sb = new StringBuilder();
+        final StringBuilder resultASCII = new StringBuilder();
```

Comment:
I'd suggest `resultAscii` to actually follow Java naming conventions.

Reply:
> I'd suggest `resultAscii` to actually follow Java naming conventions.

fixed.